conf/application.conf

```diff
@@ -64,6 +65,29 @@ maxDifferentEmails = 10
 
 passcodeExpiryMinutes = 15
 
+verifiedEmailRepo {
+    hashKey = somehashkey
+    replaceIndex = false # If true rebuilds indexes on startup
+    ttlDays = 2555 # 2555 days = 7 years
+}
+
+// "old", "new" or "both" (if both chosen but email only found in one of them a warning is logged)
+// set to "old" during migration, then "both", then "new" if no discrepencies found (i.e. ready to remove the old collection)
+verifiedEmailCheckCollection = "old"
+
+// "both" or "new"
+// set to both initially, then switch to new once migration completed (i.e. ready to remove the old collection)
+verifiedEmailUpdateCollection = "both"
+
+#GG-6759 remove following email -> hashed email collection migration
+emailMigration {
+    enabled = false
+    startAfterMinutes = 1
+    batchSize = 1000
+    batchDelayMillis = 500
+    maxDurationSeconds = 3600
```

Comment:
Let's check the maths together, inc. maxDuration, as it'll be a huge collection

Reply:
Yep currently 2.264 million verified emails in the collection. If it takes 1 second (including batch delay) per 1000 records, thats 2264 seconds or 37.66 minutes, so I've got max duration cut off at 1 hour.